apps/meteor/app/apps/server/bridges/rooms.ts

```diff
@@ -169,4 +169,26 @@ export class AppRoomBridge extends RoomBridge {
 
 		return rid;
 	}
+
+	protected getModerators(roomId: string, appId: string): Promise<IUser[]> {
+		this.orch.debugLog(`The App ${appId} is getting room moderators for room id: ${roomId}`);
+		return this.getUsersByRoomIdAndSubscriptionRole(roomId, 'moderator');
+	}
+
+	protected getOwners(roomId: string, appId: string): Promise<IUser[]> {
+		this.orch.debugLog(`The App ${appId} is getting room owners for room id: ${roomId}`);
+		return this.getUsersByRoomIdAndSubscriptionRole(roomId, 'owner');
+	}
+
+	protected getLeaders(roomId: string, appId: string): Promise<IUser[]> {
+		this.orch.debugLog(`The App ${appId} is getting room leaders for room id: ${roomId}`);
+		return this.getUsersByRoomIdAndSubscriptionRole(roomId, 'leader');
+	}
+
+	private async getUsersByRoomIdAndSubscriptionRole(roomId: string, role: string): Promise<IUser[]> {
```

Comment:
I'm thinking whether to expose this too (with a diff name) - for custom roles.

Reply:
Not sure I follow, but we can discuss it as an improvement for next release :)